File: lambdas/python_lambda/handler.py
import json
import boto3

dynamo_client = boto3.client('dynamodb')

def lambda_handler(event, context):
    
    response = dynamo_client.list_tables(
        Limit=100
    )
    table_names = response['TableNames']


    return {
        'statusCode': 200,
        'body': json.dumps({
            "tables": table_names
        })
    }

File: lambdas/node_lambda/handler.ts
import { Handler } from 'aws-lambda';
import { ListTablesCommand, DynamoDBClient } from "@aws-sdk/client-dynamodb";


const dynamoClient = new DynamoDBClient();

export const handler: Handler = async (event, context) => {
    const command = new ListTablesCommand({
        Limit: 100
    });

    const response = await dynamoClient.send(command);
    console.log(response.TableNames);
    const tableNames = response.TableNames;
    return {
        statusCode: 200,
        body: JSON.stringify({
            "tables": tableNames
        }),
    }
};